src/go/rpk/pkg/serde/embed/BUILD

```diff
@@ -0,0 +1,38 @@
+load("@rules_go//go:def.bzl", "go_library", "go_test")
+
+go_library(
+    name = "embed",
+    srcs = ["embed.go"],
+    embedsrcs = [
+        "protobuf/confluent/meta.proto",
+        "protobuf/confluent/types/decimal.proto",
+        "protobuf/google/type/README.md",
```

Comment:
does this need to be embedded?

Reply:
Good call, I've changed the Go directive to only include `.proto` files. :+1: